### DISCUSSION TITLE
List all directories which has "dev" enabled?

### DISCUSSION THREAD

**Discussion by fredrikaverpil** [pkgxdev/pkgx#866] [Help] [answered]
Can I somehow get an overview of which directories have `dev` enabled?

**Comment by mxcl** [comment, marked as answer]
```sh
cd $HOME/Library/Application\ Support/pkgx/dev && find . -name dev.pkgx.activated | xargs dirname
```

**Reply by mxcl** [reply]
obv. would be great to have a command off of `dev` to do this. It's still early days for the `dev` tool. We will get to it.

**Reply by fredrikaverpil** [reply]
For sure. But this works for now 😄 

Many thanks for this!
